### ISSUE TITLE
Stuck wp cli for Installing updating Wordpress, plugins, and themes

### ISSUE THREAD

**Issue by donna-at-keyboard** [GoogleCloudPlatform/php-docs-samples#606]
[When trying to update a Wordpress installation or the plugins on my local machine][manual], the command just stays stuck without an error message. 

The only thing I get is the connection message from the mysql connection which has been initiated through the `cloud_sql_proxy` binary in the background
ex.
~~~
donna@desktop:~/php-docs-samples/appengine/wordpress/my-wordpress-project$ vendor/bin/wp core update --path=wordpress
2018/05/13 19:55:29 Client closed local connection on 127.0.0.1:3306

~~~

But when I check the version of wordpress with the path specified I get an error, instead leaving out the path will return the wordpress version

~~~
donna@desktop:~/php-docs-samples/appengine/wordpress/my-wordpress-project$ ./vendor/bin/wp core version
4.9.5
donna@desktop:~/php-docs-samples/appengine/wordpress/my-wordpress-project$ ./vendor/bin/wp core version --path=worpress
Error: This does not seem to be a WordPress install.
Pass --path=`path/to/wordpress` or run `wp core download`.

~~~

My directory structure for the folder `my-wordpress-project` looks like this:

~~~
-rw-r-----  1 donna donna   1361 May 13 18:33 app.yaml
-rw-r-----  1 donna donna    107 May 13 18:33 composer.json
-rw-rw-r--  1 donna donna 114103 May 13 18:46 composer.lock
-rw-r-----  1 donna donna     93 May 13 18:33 cron.yaml
-rw-r-----  1 donna donna    206 May 13 18:33 php.ini
drwxrwxr-x 14 donna donna   4096 May 13 18:46 vendor
drwxrwxr-x  5 donna donna   4096 May 13 19:29 wordpress
-rw-r-----  1 donna donna     16 May 13 18:33 wp-cli.yml
~~~

How do I troubleshoot this problem or get the command out of it's stuck state?

Note: My database is in another region than the Wordpress deployment

[manual]: https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/appengine/wordpress#installupdate-wordpress-plugins-and-themes

**Comment by maxpaynestory** [comment]
Can you make both cloud sql and google app engine inside the same region.

**Comment by iotmani** [comment]
Perhaps try without the ``--path=wordpress`` parameter? My situation was slightly different but this resolved it.
If you get the same `Client closed local connection on 127.0.0.1:3306` error, I suspect the cloud proxy can't make proper connection...

**Comment by bshaffer** [comment]
So, for your particular installation, wordpress is being installed in the directory `wordpress` rather than the root of your application, so @iotmani is correct that you would need to pass `--path=wordpress` to the `vendor/bin/wp` command.

The WordPress tool has since been updated to install wordpress in the root directory for App Engine Standard (PHP 7.2).
